deepchem/models/torch_models/text_cnn.py

```diff
@@ -179,3 +186,210 @@ def forward(self, input: OneOrMany[torch.Tensor]) -> List[Any]:
             output = output.view(-1, self.n_tasks, 1)
             outputs = [output]
         return outputs
+
+
+class TextCNNModel(TorchModel):
```

Comment:
Let's add this to the docs/

Reply:
@shreyasvinaya I have a small doubt regarding updating docs. There is already an entry in the documentation for the TextCNNModel(https://deepchem.readthedocs.io/en/latest/api_reference/models.html#textcnnmodel), which currently links back to the TF implementation, should I replace it with this Torch version.

Reply:
Also do I replace Keras with PyTorch in this table(https://deepchem.readthedocs.io/en/latest/api_reference/models.html#id134) as well? I don't see any entries of Keras/PyTorch

Reply:
add this to the docs under `torch_models`

Reply:
Added to docs.